### PR TITLE
Lower requirements versions

### PR DESCRIPTION
**Pull request by a-r-r-o-w**
Lowers the requirements a bit to make it more friendly with existing user environments

### CODE REVIEW

**Review by zRzRzRzRzRzRzR — COMMENTED** [review]
"I believe there are significant changes in numpy 2.X. Should we pin the version to 1.26.4, which is the last version of numpy 1.x? However, I am not sure if there will be compatibility issues between these two versions."

File: requirements.txt [hunk 1]
```diff
@@ -1,16 +1,16 @@
-accelerate>=1.0.0
-bitsandbytes>=0.44.1
-diffusers>=0.30.4
+accelerate>=0.34.2
+bitsandbytes>=0.43.3
```
Comment (sayakpaul) [inline requirements.txt:2]: I think we can definitely unpin the version. It's safe.

File: requirements.txt [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
+diffusers>=0.30.3
 transformers>=4.45.2
-huggingface_hub>=0.25.2
+huggingface_hub>=0.24.6
```
Comment (sayakpaul) [inline requirements.txt:5]: Same here.

File: requirements.txt [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
-pandas>=2.4.1
+wandb>=0.17.7
+pandas>=2.2.2
 torch>=2.4.0
 torchvision>=0.19.0
 torchao>=0.5.0
 sentencepiece>=0.2.0
 imageio-ffmpeg>=0.5.1
-numpy>=2.1.2
+numpy>=1.26.4
```
Comment (sayakpaul) [inline requirements.txt:16]: Good one.

File: requirements.txt [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
-huggingface_hub>=0.25.2
+huggingface_hub>=0.24.6
 hf_transfer>=0.1.8
-peft>=0.13.1
+peft>=0.12.0
 decord>=0.6.0
-wandb>=0.18.3
-pandas>=2.4.1
+wandb>=0.17.7
+pandas>=2.2.2
```
Comment (sayakpaul) [inline requirements.txt:10]: Same here.

**Review by sayakpaul — APPROVED** [review]
Thanks! Just a single comment.

**Comment by a-r-r-o-w** [comment]
> I believe there are significant changes in numpy 2.X. Should we pin the version to 1.26.4, which is the last version of numpy 1.x? However, I am not sure if there will be compatibility issues between these two versions.

I have tried experimenting with both versions so far and not found any problems. I think we use very basic functionalities, and do most of things in torch land, so we should be good hopefully with this 1.26.4 version. Please let me know if you find any problems if you'd like to try

File: requirements.txt (hunk 1 above)
Comment (a-r-r-o-w) [reply requirements.txt:2]: Sounds good!

Comment (a-r-r-o-w) [inline requirements.txt:2]:
```suggestion
accelerate
bitsandbytes
```

File: requirements.txt (hunk 2 above)
Comment (a-r-r-o-w) [inline requirements.txt:5]:
```suggestion
huggingface_hub
```

File: requirements.txt (hunk 4 above)
Comment (a-r-r-o-w) [inline requirements.txt:10]:
```suggestion
wandb
pandas
```

**Comment by a-r-r-o-w** [comment]
Made some updates based on reviews, please approve if okay and merge

**Pull request merged**